packages/services/hmi-server/src/main/resources/application.properties

```diff
@@ -23,7 +23,7 @@ mp.openapi.extensions.smallrye.info.description=REST endpoints for the TERArium
 ########################################################################################################################
 data-service/mp-rest/url=http://localhost:3020
 # uncomment to link to actual dataservice
-# data-service/mp-rest/url=http://localhost:8001
+data-service/mp-rest/url=http://localhost:8001
```

Comment:
can you revert this please? You have data-service doubly defined.

Reply:
I think we want to use the real data service now?

Reply:
I can comment out the other one.

Reply:
the version on main is 3020, is I will revert to that for now.